tests/series-format-patch-bb.ref

```diff
@@ -13,4 +13,7 @@ SRC_URI += "\
 SRC_URI += "\
   file://patches/0101-b1.patch \
   "
+UMPF_BASE = "base"
+UMPF_VERSION = "20221209-1"
+PV = "${UMPF_BASE}-${UMPF_VERSION}"
```

Comment:
Why not `?=`? That would make it work independent of the ordering in the recipe?

Reply:
`meta/conf/bitbake.conf` sets

```
PV = "${@bb.parse.vars_from_file(d.getVar('FILE', False),d)[1] or '1.0'}"
```

So it would just never work (independent of the ordering) :smirk:

Reply:
Ah, so that would need to change first.